roles/privacyidea/templates/privacyidea_data_import_and_cleanup.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+# Define the MySQL credentials and database
+
+DB_USER="{{ mariadb_user }}"
+DB_NAME="{{ privacyidea_mariadb_name }}"
+DB_PASSWORD="{{ privacyidea_db_user_password }}"
+DUMP_FILE="CRQ000002489570_dump.sql"
```

Comment:
The dump file location and the token count should be in Ansible variables

Reply:
is now set via var